components/camera/LicencePlateCameraBottomSheet.tsx

```diff
@@ -63,6 +63,7 @@ const LicencePlateCameraBottomSheet = ({
               className="h-auto text-center font-source-500medium text-[62px]"
               accessibilityLabel={t('scanLicencePlate.licencePlate')}
               value={licencePlate}
+              autoCapitalize="characters"
```

Comment:
It was not here? :D :D 

I remember, that even we used this setting, sometimes it didn't capitalise som letters, so we set `.toUpperCase()` also in `onChangeText` handler.
With these two things, it's definitely better than before :D

Reply:
the .toUpperCase() was there before, but i've removed it because of bug it caused with copying the prev letters multiple times when using numbers with letters